docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -504,11 +500,6 @@ Make sure all the Pods have been terminated before you move on to the next step.
 Check the Alluxio upgrade guide page for whether the Alluxio master journal has to be formatted.
 If no format is needed, you are ready to skip the rest of this section and move on to restart all Alluxio master and worker Pods.
 
-How the journal should be formatted depends on the Alluxio master [journal type]({{ '/en/operation/Journal.html#ufs-journal-vs-embedded-journal' | relativize_url }}).
-
-*UFS Journal*
-
-If you are running UFS journal, there is only one place for the journal.
```

Comment:
Now two Journal options will both use the job to format journal, so no need to have them separated.